mount_config.go

```diff
@@ -174,6 +174,10 @@ type MountConfig struct {
 	// /proc/mounts will show the filesystem type as fuse.<Subtype>.
 	// If not set, /proc/mounts will show the filesystem type as fuse/fuseblk.
 	Subtype string
+
+	//Flag to disable async reads that are received from
+	//the kernel
+	EnableAsyncReads bool
```

Comment:
The field is named EnableAsyncReads, but the comment says “Flag to disable”. Update the comment please.